lib/org/chromium/webidl/URLExtractor.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'URLExtractor',
+
+  requires: ['org.chromium.webidl.IDLFileContents'],
+
+  properties: [
+    {
+      name: 'exclude',
+      // documentation: 'Excludes all URLs matching this array of expressions.',
```

Comment:
No need to comment this out. Also, what are `expressions` here? Regular expressions?